app/models.py

```diff
@@ -397,8 +397,8 @@ class Service(db.Model, Versioned):
     go_live_user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=True)
     go_live_user = db.relationship('User', foreign_keys=[go_live_user_id])
     go_live_at = db.Column(db.DateTime, nullable=True)
-    sending_domain = db.Column(db.String(255), nullable=True, unique=False)
-    smtp_user = db.Column(db.String(255), nullable=True, unique=False)
+    sending_domain = db.Column(db.Text(255), nullable=True, unique=False)
+    smtp_user = db.Column(db.Text(255), nullable=True, unique=False)
```

Comment:
What is the intent here?  From the attribute names, I would expect [string](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.String) to be correct.  `nullable` and `unique` are set to the default values.  I would remove them if we're changing the code for [Text](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Text).

Reply:
As this changes are co-authorized, I am not sure who changed this line, I will check it now. @kalbfled Can you give me some insights why output is having altering column from `String` to `Text` or from `Text` to String`